CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.4 Beta 2 
+
+### Additions and Improvements 
+
+- Enable TLS for JSON-RPC HTTP Service [\#253](https://github.com/hyperledger/besu/pull/253)
+
+Exposes =command line parameters to enable TLS on Ethereum JSON-RPC HTTP interface to allow clients like EthSigner to connect via TLS: 
+`--rpc-http-tls-enabled=true`
+(Optional - Only required if `--rpc-http-enabled` is set to true) Set to `true` to enable TLS. False by default.
+`--rpc-http-tls-keystore-file="/path/to/cert.pfx"`
+(Must be specified if TLS is enabled) Path to PKCS12 format key store which contains server's certificate and it's private key
+`--rpc-http-tls-keystore-password-file="/path/to/cert.passwd"`
+(Must be specified if TLS is enabled) Path to the text file containing password for unlocking key store.
+`--rpc-http-tls-known-clients-file="/path/to/rpc_tls_clients.txt"`
+(Optional) Path to a plain text file containing space separated client’s certificate’s common name and its sha-256 fingerprints when 
+they are not signed by a known CA. The presence of this file (even empty) enables TLS client authentication. That is, the client 
+presents the certificate to server on TLS handshake and server establishes that the client certificate is either signed by a 
+proper/known CA. Otherwise, server trusts client certificate by reading the sha-256 fingerprint from known clients file specified above. 
+
```

Comment:
The formating doesn't look right...

Reply:
_Pretty sure_ the wrapping should look OK when merged.

Reply:
The wrapping will look like it does on https://github.com/hyperledger/besu/blob/07fb4fe164e5eaaf105e3c4a5e85f7c03c82e032/CHANGELOG.md

Reply:
And that doesnt look right